src/components/App.js

```diff
@@ -39,19 +32,18 @@ class App extends Component {
 
   selectMovie = (id) => {
     this.setState({ movieIsSelected: true })
-    fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
-      .then(results => results.json())
-      .then(data =>
-        this.setState({
-          selectedMovie: data.movie
+      fetchSingleMovie(id)
```

Comment:
Same here - the fetch call gives you LOTS of data - if you're not using it, trim it down before setting to state.